docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/cinema_service
```

Comment:
Mounting the entire project directory into the container can lead to potential issues with file permissions and can include unnecessary files. Consider specifying only the necessary directories or files to be mounted.